student.rb

```diff
@@ -1,11 +1,18 @@
 require './person'
 
 class Student < Person
-  def initialize(age, name = 'Unknown', parent_permission: true)
+  attr_reader :classroom
+
+  def initialize(age = 0, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
   end
```

Comment:
- [ ] The error mentioned by the previous reviewer is due to your initialize method and how you're passing the `parent_permission` parameter. Notice that in your `Person` class you're receiving this `parent_permission` as a hash, so when you call `super()` in your `Student` initialize method, it sends a normal parameter but expects a hashed one:

```suggestion
  def initialize(age = 0, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @classroom = classroom
  end
```

That should get rid of the error, cheers! 🤓 🔧